### DISCUSSION TITLE
Checkbox Max displayed tags based on the width of the input

### DISCUSSION THREAD

**Discussion by acetinick** [orchidjs/tom-select#35] [Ideas]
**Is your feature request related to a problem? Please describe.**

max displayed tags using the width of the input.  Really nice to have when using a small input, for things like search filtering etc.

![image](https://user-images.githubusercontent.com/571649/108808422-460e4900-75af-11eb-9555-d348468d9aea.png)

**Describe the solution you'd like**
- ability to just show options that dont fit as comma separated, with "...." and displayed number.
- checkboxes selections for each option

**Describe alternatives you've considered**
similar to this example
https://rsuitejs.com/components/check-picker/

**Comment by oyejorge** [comment]
I like it!

**Comment by oyejorge** [comment]
It's still buggy, [but here's what it might look like](https://jsfiddle.net/a5eo3m90/)

**Comment by acetinick** [comment]
This is awesome start) looks good for us to use! Thanks.

by the way, we are looking to use this plugin more extensively throughout our application, would you be interested in paid development to help enhance this project for our project? If so, how could I contact you?

**Comment by oyejorge** [comment]
I wouldn't use it yet. Removing selections works, but it conflicts with the default for selecting options. It will work better when set up as a plugin.

Also, thanks, let's chat. Would it work if I send you a message on peopleforce.io?

**Comment by acetinick** [comment]
Yep of course just reach out to me at andrew@peopleforce.io

**Comment by oyejorge** [comment]
[Here's some progress I've been meaning to push out](https://github.com/orchidjs/tom-select/tree/1.3/src/plugins/checkbox_options). It's a bit different from the example above but, as far as I've tested, works pretty well.

**Comment by oyejorge** [comment]
Just pushed out v1.3 with [the checkbox_options plugin](https://tom-select.js.org/plugins/checkbox-options/)

**Comment by mmarton** [comment]
Hi!

is there any update regarding this feature? Will it or some alternative be part of 2.0?

**Comment by dehy** [comment]
This feature as displayed in the picture in the original post would be awesome. Currently, the input grows in height and thus may not be ideal for inline forms.

<img width="1322" alt="Capture d’écran 2021-12-03 à 11 01 05" src="https://user-images.githubusercontent.com/1552768/144584302-60105f05-fd3c-41e3-90ef-f332d5b08ceb.png">

Do you need some help on this? How did you consider this? Another plugin? an update of the checkbox_options plugin?

thanks!

**Reply by oyejorge** [reply]
[I think the remaining piece is this](https://github.com/orchidjs/tom-select/discussions/183#discussioncomment-1747921)
